tests/conftest.py

```diff
@@ -33,6 +34,36 @@
 IS_LINUX = sys.platform.startswith("linux")
 
 
+@pytest.fixture(autouse=True)
+def blockbuster(request: pytest.FixtureRequest) -> Iterator[None]:
+    # No blockbuster for benchmark tests.
+    node = request.node.parent
+    while node:
+        if node.name.startswith("test_benchmarks"):
+            yield
+            return
+        node = node.parent
+    with blockbuster_ctx(
+        "aiohttp", excluded_modules=["aiohttp.pytest_plugin", "aiohttp.test_utils"]
+    ) as bb:
+        # TODO: Fix blocking call in ClientRequest's constructor.
+        # https://github.com/aio-libs/aiohttp/issues/10435
+        for func in ["io.TextIOWrapper.read", "os.stat"]:
+            bb.functions[func].can_block_in("aiohttp/client_reqrep.py", "update_auth")
+        for func in [
+            "os.getcwd",
+            "os.readlink",
+            "os.stat",
+            "os.path.abspath",
+            "os.path.samestat",
+        ]:
+            bb.functions[func].can_block_in(
+                "aiohttp/web_urldispatcher.py", "add_static"
+            )
+        bb.functions["os.getcwd"].can_block_in("coverage/control.py", "_should_trace")
```

Comment:
NOTE: not needed anymore with blockbuster v1.5.22+

Reply:
Would you please send another PR. We can merge on top and than backport once we get the backport PR merged.

Reply:
sure, no problem.

Reply:
Thanks!